io_export_objex2/properties.py

```diff
@@ -68,19 +69,60 @@ class ObjexSceneProperties(bpy.types.PropertyGroup):
         default={'BLENDER_TO_OBJEX','OBJEX_TO_BLENDER'},
     )
 
+    mode_menu = bpy.props.EnumProperty(
+        items=[
+            ('menu_mode_combiner',    'Combiner', '(A-B)*C+D',                'SHADING_RENDERED', 0),
+            ('menu_mode_texture',     'Texture',  'Texture',                  'TEXTURE',          1),
+            ('menu_mode_render',      'F3D',      'F3DZEX2 Flags',            'RENDERLAYERS',     2),
+            ('menu_mode_settings',    'Settings', 'Objex2 Material Settings', 'TOOL_SETTINGS',    3),
+        ],
+        name='Menu Mode',
+        default='menu_mode_texture',
+    )
+
     write_primitive_color = bpy.props.BoolProperty(
-            name='Set prim color (global)',
+            name='Set prim (global)',
```

Comment:
This prop is drawn in scene properties, I think keeping "color" is useful as there's otherwise not enough context besides "it's an objex2 thing"

Reply:
Prim and Env are only used when talking about color so I'm not sure if that's completely necessary. Especially as how I layed them, they mostly get `...` by the end as how long they are. Maybe the hint could use the explatanion for them in this case?

But if you insist, I'll change them.

Reply:
Returned these as they were